### ISSUE TITLE
[BUG] New tab is not working for Edge 

### ISSUE THREAD

**Issue by madhuriinakollu** [microsoft/playwright#15108]
**Context:**
- Playwright Version: [1.22.1]
- Operating System: [Windows]
- Node.js version: [e.g. 10.0.0]
- Browser: [Edge]
- Extra: [Enabled the IE explorer mode]

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```TypeSciript
import { test, expect,chromium } from '@playwright/test';

test.only('test@Reg', async () => {

    var localAppDataPath = "C://Users//<username>//AppData//Local//Microsoft//Edge//User Data";
    const context  = await chromium.launchPersistentContext(localAppDataPath ,{
        channel: 'msedge',
        headless: false,
        slowMo: 10,
        ignoreDefaultArgs: ['--remote-debugging-pipe'],
        executablePath: "C://Program Files (x86)//Microsoft//Edge//Application//msedge.exe",
    })  
      const page = await context.newPage()
      await page.goto('https://www.google.com');
```

**Describe the bug**

Not able to open new tab with ignoreDefaultArgs: ['--remote-debugging-pipe'], option.  this code is just open the browser and exit as the timeout.

Basically, I need to load the page to turn off remote debugging.

Thanks,
Madhuri

**Comment by mxschmitt** [comment]
Playwright needs to communicate with the browser, this is based on the pipe, when you disable that arg, it does not work.

Closing by that since its expected.
